src/GradhSph/GradhSphTree.cpp

```diff
@@ -592,7 +592,11 @@ void GradhSphTree<ndim,ParticleType>::UpdateAllSphForces
         ComputeFastQuadrupoleForces(Nactive, Ngravcell, gravcell, cell, activepart, sph->types);
       }
 
-
+      // Set gpot_hydro (RadWS) before sink contribution
+      // TODO (MERCER): Add gpot contribution from sinks that form
```

Comment:
Is this something that needs doing now before the merge or just something on the to-do list for the future?

Reply:
Had a chat with Dimitris, it's safe to ignore all sink gpot so this is fine as is, I'll remove the TODO.